forms-html/yazen_lawin/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+
+<body>
+<div style="font-weight: bold;">
+
+
+
```

Comment:
Don't leave more than one new line for each section of code 
You left e lines here on line 6 and 7 and 8.